lib/t_web/views/view_helpers.ex

```diff
@@ -69,7 +69,19 @@ defmodule TWeb.ViewHelpers do
           |> Enum.reduce([], fn label, acc ->
             case label do
               %{"s3_key" => key, "question" => "audio"} ->
-                label = Map.put(label, "url", Accounts.media_url(key))
+                label = Map.put(label, "url", media_cdn_url(key))
+                [label | acc]
+
+              %{
+                "s3_key" => key,
+                "placeholder_s3_key" => placeholder_s3_key,
```

Comment:
`placeholder_s3_key` -> `s3_key`, `s3_key` -> `video_s3_key` to be consistent with video backgrounds?

Reply:
right